am-lib/src/main/java/uk/gov/hmcts/reform/amlib/models/ExplicitPermissions.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.reform.amlib.models;
+
+import uk.gov.hmcts.reform.amlib.enums.Permissions;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class ExplicitPermissions {
+    private final List<Permissions> userPermissions;
+
+    public ExplicitPermissions(List<Permissions> userPermissions) {
+        this.userPermissions = userPermissions;
```

Comment:
Maybe `Set<Permissions>` would be better here?

Now this can be done like that:
```
new ExplicitPermission(asList(READ, READ));
```

yes, it doens't make sens, but when we sum it up we get 4 (UPDATE) instead of 2 (READ). If we have Set it wouldn't happen.